Shaders/RHI/TrianglesConstantBuffer.azsl

```diff
@@ -28,7 +28,8 @@ struct InstanceInfo
     // NOTE: This actually shouldn't be required, but the current validation will give a false positive
     // when the stride isn't the same size as the stride defined on the applicaiton side.
     // This assumes 256 alignment, which is standard for DX12, but is variable in Vulkan
-    float padding0[44]; 
+    float m_padding0[11];
+    float3 m_padding1;
```

Comment:
float arrays in struct is not a good idea. The memory layout for float padding0[11] becomes
(padding0[0].x, empty, empty, empty)
(padding0[1].x, empty, empty, empty)
(padding0[2].x, empty, empty, empty)
(padding0[3].x, empty, empty, empty)
..
(padding0[10].x)
which is super confusing. Just do this instead

float4 padding0[11];
and it should work

Reply:
Changed